reana_workflow_engine_serial/tasks.py

```diff
@@ -117,6 +117,7 @@ def run_step(
             unpacked_image=step.get("unpacked_image", False),
             kubernetes_uid=step.get("kubernetes_uid", None),
             kubernetes_memory_limit=step.get("kubernetes_memory_limit", None),
+            kubernetes_job_timeout=step.get("kubernetes_job_timeout", None),
```

Comment:
`None` is the default backup value, it shouldn't be needed. The same applies to other options.

Reply:
Good catch. Done